spharpy/plot/_utils.py

```diff
@@ -0,0 +1,82 @@
+"""Private utility functions for plot module."""
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def _add_colorbar(colorbar, fig, ax, mappable, label):
+    """
+    Add colorbar to plot.
+
+    Parameters
+    ----------
+    colorbar : bool
+        Flag indicating if a colobar should be added to the plot
+    fig : matplotlib.figure.Figure
+        Figure to plot on.
+    ax : list[matplotlib.axes.Axes]
+        Either a list of two axes objects or a list with one axis and None
+        object.
+    mappable : matplotlib.cm.ScalarMappable
+        The matplotlib.cm.ScalarMappable described by the colorbar.
+    label : string
+        colorbar label
+
+    Returns
+    -------
+    cb : matplotlib.colorbar.Colorbar
+        Returns matplotlib colorbar object.
+    """
+    if colorbar:
+        if ax[1] is None:
+            cb = fig.colorbar(mappable, ax=ax[0])
+        else:
+            cb = fig.colorbar(mappable, ax=ax[1])
+        cb.set_label(label)
+    else:
+        cb = None
+
+    return cb
+
+
+def _prepare_plot(ax, projection=None):
+    """
+    Returns a figure to plot on.
+
+    Parameters
+    ----------
+    ax : matplotlib.axes.Axes or list, tuple or ndarray of maplotlib.axes.Axes
+        Axes to plot on. The default is None in which case the axes are
```

Comment:
Do I get it right, that this means the default is calling it as `_prepare_plot()`? Then the default for ax is missing in line 42.